src/main/java/br/vet/certvet/dto/requests/AgendamentoRequestDto.java

```diff
@@ -1,21 +1,29 @@
 package br.vet.certvet.dto.requests;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
 import javax.validation.constraints.Future;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 import java.time.LocalDateTime;
 
+@Getter
 public class AgendamentoRequestDto {
     @NotNull(message = "Selecione um animal")
-    public Long animal;
+    protected Long animal;
```

Comment:
```suggestion
    private Long animal;
```

Normalmente, o uso de protected é bem específico para ser usado em classes que sofrerão override. Essa, eu entendo que seja candidata a se tornar `final`. será que não seria melhor tornar `private` ou tornar a classe `record`?